app/models/manageiq/providers/amazon/storage_manager/ebs/cloud_volume.rb

```diff
@@ -100,8 +100,9 @@ def provider_object(connection = nil)
 
   def modify_volume_options(options = {})
     modify_opts = {}
+
     if volume_type != 'standard'
-      modify_opts[:volume_type] = options[:volume_type] if options[:volume_type] && options[:volume_type] != 'standard'
+      modify_opts[:volume_type] = options[:volume_type] if options[:volume_type] && options[:volume_type] != volume_type
       modify_opts[:size]        = Integer(options[:size]) if options[:size] && Integer(options[:size]).gigabytes != size
```

Comment:
You are correct; I wondered about this check if it makes sense or not here. The UI prevents setting standard while editing. On the other hand, when updating the volume directly through the provider, it may be better to actually get an exception if you tried to set `standard` (and it wasn't `standard` before).

For example, if I have a `gp2` volume and call `update_volume` with new `options[:volume_type] = 'standard'`, it may be better to let this through and let AWS exception.

What do you think? I can add the check for `standard` again.